filters/openpolicyagent/response.go

```diff
@@ -0,0 +1,77 @@
+package openpolicyagent
+
+import (
+	"bytes"
+	"io"
+
+	"net/http"
+
+	"github.com/open-policy-agent/opa-envoy-plugin/envoyauth"
+	"github.com/opentracing/opentracing-go"
+	"github.com/zalando/skipper/filters"
+)
+
+func (opa *OpenPolicyAgentInstance) RejectInvalidDecisionError(fc filters.FilterContext, span opentracing.Span, result *envoyauth.EvalResult, err error) {
+	resp := http.Response{}
+
+	fc.Metrics().IncCounter(opa.MetricsKey("decision.err"))
+
```

Comment:
drop this whitespace and set either first the status code xor do metrics and tracing first.